selftests/functional/basic.py

```diff
@@ -739,6 +755,64 @@ def test_store_logging_stream(self):
                 stream_line,
             )
 
+    def test_store_logging_stream_all(self):
```

Comment:
Great new isolation tests here, could we also consider one for a potential stream from an external library? For instance, let's say that some tests use `numpy` or some helper framework for the avocado tests. Such an isolation test could specify how the process of adding a custom stream or going about this should be.

Reply:
sure